### PR TITLE
Parameterized values

### LINKED ISSUE #81: V-92961 incorrect matcher
The expected value of inactivity limit should not equal 0 per the check text:
> Value: 0x00000384 (900) (or less, excluding \"0\" which is effectively disabled)

[The test](https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/blob/master/controls/V-92961.rb#L44) should be corrected to:
```
its('InactivityTimeoutSecs') { should_not eq 0 }
```

### PR DESCRIPTION
**Pull request by karikarshivani**
fixed #81

### CODE REVIEW

File: controls/V-92961.rb [hunk 1]
```diff
@@ -41,7 +41,7 @@
       its('InactivityTimeoutSecs') { should be <= 900 }
     end
     describe registry_key('HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System') do
-      its('InactivityTimeoutSecs') { should eq 0 }
+      its('InactivityTimeoutSecs') { should_not eq 0 }
```
Comment (karikarshivani) [inline controls/V-92961.rb:44]: Context:

> Value: 0x00000384 (900) (or less, excluding "0" which is effectively disabled)

File: controls/V-92961.rb [hunk 2]
```diff
@@ -36,13 +36,9 @@
...
-  describe.one do
-    describe registry_key('HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System') do
-      its('InactivityTimeoutSecs') { should be <= 900 }
-    end
-    describe registry_key('HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System') do
-      its('InactivityTimeoutSecs') { should eq 0 }
-    end
+  describe registry_key('HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System') do
+    its('InactivityTimeoutSecs') { should be <= 900 }
+    its('InactivityTimeoutSecs') { should_not eq 0 }
```
Comment (karikarshivani) [inline controls/V-92961.rb:41]: Context:

> Value: 0x00000384 (900) (or less, excluding "0" which is effectively disabled)

**Comment by ejaronne** [comment]
V-93461 inadvertently adds an additional skip message and also doesn't show the input value in the proper skip, having something to do with the way the input is used within the skip statement:

> microsoft-windows-server-2019-stig-baseline> inspec exec . --controls V-93461                                                              
> Profile:   microsoft-windows-server-2019-stig-baseline (microsoft-windows-server-2019-stig-baseline)
> Version:   1.3.22
> Target:    local://
> Target ID: 648fe408-2341-46c1-82a3-20730cdeee10
> 
>   [SKIP]  V-93461: Windows Server 2019 manually managed application account passwords must be at least 15 characters in length. (2 skipped)
>      [SKIP]  minimum_password_length_manual
>      [SKIP]  Determine if manually managed application/service accounts exist. If none exist, this is NA. Verify the organization has a policy to ensure passwords for manually managed application/service accounts are at least #<RSpec::Core::Example "minimum_password_length_manual"> characters in length.
> 
> 
> Profile Summary: 0 successful controls, 0 control failures, 1 control skipped
> Test Summary: 0 successful, 0 failures, 2 skipped

Otherwise all looks okay.

**Review by ejaronne — APPROVED** [review]
Looks good thanks!

**Pull request merged**